doc/lightning_tensor/device.rst

```diff
@@ -19,12 +19,12 @@ The default setup for the MPS tensor network approximation is:
 The ``lightning.tensor`` device dispatches all operations to be performed on a CUDA-capable GPU of generation SM 7.0 (Volta)
```

Comment:
> ``cutoff`` (singular value truncation threshold) defaults to ``0`` .

We are currently not actually using this, right? Perhaps worth a small comment that this is actually not supported yet

Reply:
Or was it just for MPOs where this isnt used?

Reply:
> ``cutoff_mode`` (singular value truncation mode) defaults to ``abs`` , considering the absolute values of the singular values; Alternatively, users can opt to set ``cutoff_mode`` to ``rel`` to consider the relative values of the singular values. 

Singular values are always non-negative and real, why do we need this at all?

Reply:
> >cutoff (singular value truncation threshold) defaults to 0 .

> We are currently not actually using this, right? Perhaps worth a small comment that this is actually not supported yet

``cutoff`` is supported by the ``lightning.tensor`` device. A short description on how the ``cutensornet`` backend reduce the bond extent dimensions is added.

Reply:
> Or was it just for MPOs where this isnt used?
It is for both `MPS` and `MPO` in the `cutensornet` context as you can find out the details in the newly added links.

Reply:
> > ``cutoff_mode`` (singular value truncation mode) defaults to ``abs`` , considering the absolute values of the singular values; Alternatively, users can opt to set ``cutoff_mode`` to ``rel`` to consider the relative values of the singular values. 

> Singular values are always non-negative and real, why do we need this at all?

This is a feature offered by ``cutensornet``. You can get more info [here](https://docs.nvidia.com/cuda/cuquantum/latest/cutensornet/api/types.html#cutensornettensorsvdconfigattributes-t).